.codecov.yml

```diff
@@ -0,0 +1,11 @@
+codecov:
+  bot: aslo
+coverage:
+  status:
+    patch:
+      default:
+        target: 0%
```

Comment:
Set a target of 0 so that these statuses will always be green, regardless of code coverage changes. Later, we can change these to use `target: auto`, the default setting, to fail builds if the coverage goes down below its current setting. 

Also note that it's an option to have these statuses never block merging, even when they are red. I still think this is valuable as-is for the time being.